generate_testdata.sh

```diff
@@ -14,14 +14,15 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 
```

Comment:
Hi @estroz,  

We need also update the e2e tests to ensure it. See: https://github.com/kubernetes-sigs/kubebuilder/blob/master/test/e2e/v3/e2e_suite.go

IMO we need to: 
- Make the tests with the v3 alpha plugin 
- Make all tests to ensure that the manager will work as expected when it is using v1 or betav1 version

Could you please do that for we are able to ensure these changes and get it merged?

Reply:
Done.